### DISCUSSION TITLE
Several Slurm controllers.

### DISCUSSION THREAD

**Discussion by ek-nag** [GoogleCloudPlatform/cluster-toolkit#2060] [Ideas and feature requests]
For the production implementations it would very useful to be able to run more than one Slurm controller node. Can we do this currently via HPC Toolkit? Here is the reference link to the relevant Slurm documentation. https://slurm.schedmd.com/slurm.conf.html#OPT_SlurmctldHost

**Comment by harshthakkar01** [comment]
Thanks for bringing this up. Slurm GCP doesn't support multiple controller nodes.

**Reply by andybubu** [reply]
@harshthakkar01 is this by change supported now. Or any plans to support it at all?
